src/views/Planning/components/AssignmentRow.tsx

```diff
@@ -108,18 +101,22 @@ const AssignmentRowContent = ({ index, onSelect }: {
   }
 
   return (
-    <div className='flex flex-col gap-2 text-sm px-6 pt-2.5 pb-4 hover:bg-muted'>
+    <div
+      onClick={(event) => {
+        if (assignmentType === 'text' || assignmentType === 'flash') {
+          onOpenArticleEvent(event)
+        } else {
```

Comment:
It might seem like a good idea to support editing photo assignments on click while opening text assignments article for editing. But it feels a bit confusing that clicking the row edit the photo assignment whereas clicking the text assignment row open the article for editing instead.

I think having a consistent behavior, even though nothing then happens when clicking the photo assignment row, is less confusing. At least I think we should discuss it more.

Reply:
Me and @hugowetterberg had that discussion yesterday. Yes, different outcome for the same action, however we both felt that it's the "primary" action for each, open vs edit.